### PR TITLE
feat: Added very simple individual add-on page

### PR DESCRIPTION
**Pull request by zebby09**
- Added react-router-dom dependency
- Implemented routing to individual add-on page
- Created simple add-on page including:
  -  Name
  - TODO: Author
  - Summary
  - TODO: Install button
  - TODO: ReadMe
- Moved storybook decorators to component level 
- Added a test util to return a store with preloaded mock addons
- Wrote routing tests

### CODE REVIEW

File: src/style.css
Comment (Tjalle-S) [inline src/style.css:1]: Waarom deze changes? Is dit van Prettier?

**Review by Tjalle-S — CHANGES_REQUESTED** [review]
I have added a couple of comments. Otherwise, everything seems good.

File: src/style.css
Comment (zebby09) [reply src/style.css:1]: Ja lol, had niet gezien dat ik die ook gecommit had. Zal het ff terugzetten

File: src/pages/AddOnPage.tsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  // Retrieve URL param
+  const { id: thisId } = useParams();
+
+  // Find add-on in state using URL param
+  const thisAddOn: Addon | undefined = useAppSelector((state: RootState) =>
+    state.addons.allAddOns.find(addon => addon.id === thisId)
+  );
+
+  // Render simple error page when URL param not found in state
+  if (!thisAddOn) {
```
[the remark this answers is not in the document]
Comment (zebby09) [reply src/pages/AddOnPage.tsx:29]: Is dit goed? `if (thisAddOn === undefined) {`

**Review by stickyPiston — APPROVED** [review]
頑張った！

**Review by Tjalle-S — APPROVED** [review]

**Pull request merged**